.github/workflows/performance-benchmarks-naslib.yml

```
@@ -0,0 +1,197 @@
+name: Performance Benchmarks with naslib
```

A nit. https://github.com/automl/NASLib
```suggestion
name: Performance Benchmarks with NASLib
```